web/packages/teleport/src/Assist/Assist.tsx

```diff
@@ -94,8 +97,8 @@ function variables(props: { viewMode: ViewMode }) {
         '--assist-border-radius': '0',
         '--assist-left': 'auto',
         '--assist-right': '0',
-        '--assist-width': '520px',
-        '--assist-height': '100vh',
+        '--assist-width': `${dockedAssistWidth}px`,
+        '--assist-height': 'calc(100vh - 72px)',
```

Comment:
the height becomes incorrect when the topbar shrinks

<img width="1258" alt="image" src="https://github.com/gravitational/teleport/assets/43280172/2d3a9617-0ba1-4cec-b2ac-3c7d9a756179">

Reply:
I'm not sure if something was out of date, but the code suggested is already that way in the branch. perhaps you were looking at an old one?

Reply:
sorry i should've been more clear. so the assist height is hard coded `100vh - 72px`, but that only works with the largest topbar height. our topbar shrinks as screens sizes goes down, this leaves gaps at the top and the bottom of the assist (as shown by red arrows in the image, the image you took also has gaps too)